packages/ui/src/components/molecules/OrderSummary/styles.scss

```
@@ -12,6 +12,11 @@
   --fs-order-summary-total-text-size                 : var(--fs-text-size-3);
   --fs-order-summary-total-text-font-weight          : var(--fs-text-weight-bold);
 
+  // Taxes label
+  --fs-order-summary-taxes-label-color        : var(--fs-color-info-text);
+  --fs-order-summary-taxes-text-size          : var(--fs-text-size-tiny);
+  --fs-order-summary-taxes-text-weight        : var(--fs-text-weight-regular);
```

Just to keep the pattern.

```suggestion
  --fs-order-summary-taxes-label-color               : var(--fs-color-info-text);
  --fs-order-summary-taxes-text-size                 : var(--fs-text-size-tiny);
  --fs-order-summary-taxes-text-weight               : var(--fs-text-weight-regular);
```